test/deployframework/context.go

```diff
@@ -76,19 +76,7 @@ type DeployerCtx struct {
 }
 
 // NewDeployerCtx constructs and returns a new DeployerCtx object
-func (df *DeployFramework) NewDeployerCtx(
-	namespace,
-	meteringOperatorImageRepo,
-	meteringOperatorImageTag,
-	reportingOperatorImageRepo,
-	reportingOperatorImageTag,
-	catalogSourceName,
-	catalogSourceNamespace,
-	subscriptionChannel,
-	outputPath string,
-	extraLocalEnvVars []string,
-	spec metering.MeteringConfigSpec,
-) (*DeployerCtx, error) {
+func (df *DeployFramework) NewDeployerCtx(namespace, meteringOperatorImageRepo, meteringOperatorImageTag, reportingOperatorImageRepo, reportingOperatorImageTag, catalogSourceName, catalogSourceNamespace, subscriptionChannel, outputPath string, extraLocalEnvVars []string, spec metering.MeteringConfigSpec, nfsm *metering.MeteringConfig) (*DeployerCtx, error) {
```

Comment:
I would suggest to follow the format as before... I don't think PR reviewers would appreciate one long line like that.